pkg/generate/app/cmd/newapp.go

```diff
@@ -542,8 +543,36 @@ func (c *AppConfig) buildPipelines(components app.ComponentReferences, environme
 						}
 					}
 				}
-				if pipeline, err = app.NewBuildPipeline(ref.Input().String(), input, c.OutputDocker, strategy, c.GetBuildEnvironment(environment), source); err != nil {
-					return nil, fmt.Errorf("can't build %q: %v", ref.Input(), err)
+
```

Comment:
You can specify multiple pipelines today.

On Oct 19, 2015, at 8:38 AM, Rodolfo Carvalho <notifications@github.com>
wrote:

In pkg/generate/app/cmd/newapp.go
<https://github.com/openshift/origin/pull/5141#discussion_r42365386>:

> @@ -542,8 +543,36 @@ func (c *AppConfig) buildPipelines(components app.ComponentReferences, environme
>  						}
>  					}
>  				}
> -				if pipeline, err = app.NewBuildPipeline(ref.Input().String(), input, c.OutputDocker, strategy, c.GetBuildEnvironment(environment), source); err != nil {
> -					return nil, fmt.Errorf("can't build %q: %v", ref.Input(), err)
> +

is it the same for --name? What kind of use of new-app /new-build produces
2 builds?

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/5141/files#r42365386>.